From 83772ce68a8bb0ba22c188fc3230eef5cc36e501 Mon Sep 17 00:00:00 2001
From: Sridhar Kasukurthi <skasuk@codeaurora.org>
Date: Wed, 31 May 2017 21:18:03 +0530
Subject: [PATCH 11/13] MSIM: Fix to set Mcc & Mnc with correct subId

Changes to set Mcc Mnc on both the slots with correct subId

Change-Id: I36107f6c73e5532de62bb723aee278966524e3a7
CRs-Fixed: 2054858
---
 .../com/android/internal/telephony/SubscriptionInfoUpdater.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java b/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java
index 0b2c0fd0f..a205d1466 100644
--- a/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java
+++ b/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java
@@ -384,8 +384,8 @@ public class SubscriptionInfoUpdater extends Handler {
             for (int subId : subIds) {
                 TelephonyManager tm = TelephonyManager.getDefault();
 
-                String operator = tm.getSimOperatorNumeric(subId);
                 slotId = SubscriptionController.getInstance().getPhoneId(subId);
+                String operator = tm.getSimOperatorNumeric(subId);
 
                 if (!TextUtils.isEmpty(operator)) {
                     if (subId == SubscriptionController.getInstance().getDefaultSubId()) {
-- 
2.17.1

